src/mumble/AudioConfigDialog.cpp

```diff
@@ -129,6 +131,8 @@ void AudioInputDialog::load(const Settings &r) {
 	qlePushClickPathOn->setText(r.qsTxAudioCueOn);
 	qlePushClickPathOff->setText(r.qsTxAudioCueOff);
 	qleMuteCuePath->setText(r.qsTxMuteCue);
+	s.bMuteCueShown = r.bMuteCueShown;
```

Comment:
IIRC, this is redundant as the boolean in the settings object is processed by SettingsMacros when Mumble starts

The ``::load`` method is only used to apply the settings to the visual Qt elements or do other more complex loading processing. So I think your ``MuteCueShown = MuteCueShown`` can be safely removed.

Reply:
I added this line as a way to reset the mute cue popup when the user resets all of the settings to default otherwise this boolean variable will never be updated again.  This wasn't part of the request but I figured it may be more intuitive, I can remove either way though.

Thank you for all of the feedback.  I will work on implementing everything else you recommended.

Reply:
Hm? I have not tried it, but should [this line](https://github.com/mumble-voip/mumble/blob/master/src/mumble/ConfigDialog.cpp#L171) trigger the mute cue alert to be reset to the value in Settings.h Line 229? Or am I maybe missing something?

Reply:
Wait, I think you are right. Since the mute cue alert is not represented by a Qt UI element, it will not be updated by the reset. Good catch!

But this can surely not be the only setting in Mumble that has no visual representation. I wonder how this is handled, or if this is not correctly handled then :thinking: 
Keep this line for now